RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -142,20 +142,51 @@ struct SubscriptionDetailsView: View {
 
     var body: some View {
         VStack(alignment: .leading, spacing: 8) {
-            Text("\(subscriptionInformation.title) - \(subscriptionInformation.durationTitle)")
-                .font(.subheadline)
-                .padding([.horizontal, .top])
-
-            Text("\(subscriptionInformation.price)")
-                .font(.caption)
-                .foregroundColor(Color.gray)
-                .padding(.horizontal)
-
-            if let nextRenewal =  subscriptionInformation.nextRenewalString {
-                Text("\(subscriptionInformation.renewalString): \(String(describing: nextRenewal))")
+            VStack(alignment: .leading) {
+                Text("\(subscriptionInformation.title)")
+                    .font(.headline)
+                Text("\(subscriptionInformation.explanation)")
+                    .frame(maxWidth: 200, alignment: .leading)
```

Comment:
Hmm is setting a `maxWidth` necessary? Not sure if it would be better to set a percentage max width or something like that if it's necessary

Reply:
Without this `maxWidth`, this text would cause the outer card to span the entire width of the (phone) screen, which I think doesn't look great. 

I'd be okay setting it to a relative size, but I haven't found an easy way to do so. Some suggest wrapping it all in a `GeometryReader`, or using a custom `Layout`. Let me know if you know another way! 

I did check `maxWidth: 200` on an iPhone SE, and that still looks fine.